### ISSUE TITLE
Missing pre-alpha statement and a warning

### ISSUE THREAD

**Issue by thereisnotime** [bcedu/VGrive#112]
**VGrive**
This piece of software is so badly implemented that it should be called pre-alpha or something like that with a big statement in the readme "THIS IS A WIP AND IT DOES NOT WORK PROPERLY" and link to alternative solutions.

My story begins when I tried syncing one of my main folders to Google Drive. With all the default configuration it just started pushing and pushing stuff in the cloud without removing them until it filled my account there. This is when I turned on the remove feature and I left it alone for few days as it seems to be working.

Surprise - after few power outages somehow the drive files went missing and it has deleted my local files to the .trash folder. This happened two times until I found that VGrive was the reason. There is something so wrong and poorly implemented in it that it makes this software unreliable as hell. I would have troubleshooted and provided a proper bug report, but instead I had to spend 3+ hours recovering files it deleted from the .trash folder because it has no easy method of doing so and deleted everything in one root folder without keeping the structure. The only thing that belongs in this folder is VGrive. If the developers lack the knowledge or do not even bother to implement the proper Trash system or create a simple recursion when moving stuff to the .trash, at least warn the users that it can't function properly.

**TL;DR:**
- The sync method is terrible;
- The auto start does not work properly;
- No option to recover files that VGrive deleted files from .trash and leaves them in one folder;

**Comment by bcedu** [comment]
1 - There is no need to be rude
2 - LICENCE: `the GPL clearly explains
that there is no warranty for this free software`
3 - It's a program I made in my free time. It's not a software made by a full team of paid workers. I did it IN MY FREE TIME. If you don't like it, don't use it.
4 - There is no need to be rude
5 - There is no need to be rude

**Comment by marathone** [comment]
But that poster is right @bcedu . This is archive software that works on files people want to save. So therefore it has to be very reliable. You should put a warning that this is Alpha software. It's not being rude, it's being straightforward and honest. I gave up on this software when it was first mentioned. It really is a POS.

**Comment by thereisnotime** [comment]
@bcedu I lost a lot of hours because there was no warning about the reliability. You can surely understand how frustrating it is (I have not yet recovered all files). 

We all create software in Github in our free time but to trust software with your important data - there has to be some warning that it might not get the job done and possible loss of data.
